plotly/plotly/plotly.py

```diff
@@ -1271,6 +1343,40 @@ def validate_credentials(credentials):
         raise exceptions.PlotlyLocalCredentialsError()
 
 
+def add_share_key_to_url(response, **payload):
+    """
+    Update plot's url to include the secret key
+
+    payload keyword agruments:
+    un (string) -- username from credentials file
+    kwargs (dict) --
+        'world_readable': (bool) make the plot public/ private
+        'sharing': ('public' | 'private' | 'secret')
+                    set privacy for the url
+
+    """
```

Comment:
could we unpack these input arguments to make it more user-friendly? It'll be easier to test that way, and a user could actually use this function if they wanted to. In other words, take `(url, un, world_readable, sharing)` as the call signature